### PR TITLE
styled CountryCard and CountriesList

### PR DESCRIPTION
**Pull request by marcinel123**
Hi Adam,

In this PR I finished styling inside CountryCard and CountriesList components.

Please have a look and let me know.

Martin

### CODE REVIEW

File: src/components/countryCard/CountryCard.styles.ts [hunk 1]
```diff
@@ -1,10 +1,61 @@
...
+export const StyledCardFlagImage = styled.img`
+	width: 100%;
+	height: 50%;
+	object-fit: cover;
+	@media screen and (min-width: ${(props) => props.theme.breakpoints.medium}) {
+		height: 45%;
+	}
+`;
+
+export const StyledDivWithCountryInfo = styled.div`
```
Comment (AdamKniec) [inline src/components/countryCard/CountryCard.styles.ts:43]: StyledCountryInfo 👍

File: src/components/countryCard/CountryCard.tsx [hunk 2]
```diff
@@ -9,10 +15,21 @@ export const CountryCard = ({
...
-		<img src={flags.svg} alt={flags.alt} />
-		<p>{name.common}</p>
-		<p>Population:{population}</p>
-		<p>Region:{region}</p>
-		<p>Capital: {capital}</p>
+		<StyledCardFlagImage src={flags.svg} alt={flags.alt} />
+		<StyledDivWithCountryInfo>
+			<StyledCountryName>{name.common}</StyledCountryName>
+			<StyledParagraphWithCountryDetails>
+				<strong>Population: </strong>
```
Comment (AdamKniec) [inline src/components/countryCard/CountryCard.tsx]: Why did You use strong here?

**Comment by AdamKniec** [comment]
there should be 4 columns on desktop
image

i see 7

image

**Comment by AdamKniec** [comment]
Make 4 columns on desktop and make the fonts bigger in the card component. I can barely see anything :D

**Comment by AdamKniec** [comment]
Code looks good 🎖️

File: src/components/countryCard/CountryCard.styles.ts (hunk 1 above)
Comment (marcinel123) [reply src/components/countryCard/CountryCard.styles.ts:43]: thanks :)

File: src/components/countryCard/CountryCard.tsx (hunk 2 above)
Comment (marcinel123) [reply src/components/countryCard/CountryCard.tsx]: to make names like Population or Region more weight. I also used font weight in StyledParagraphWithCountryDetails but for example: Population must be bold and variable {population} also have to be semiBold. So I used "strong" to make one more bold than the other.

Comment (AdamKniec) [reply src/components/countryCard/CountryCard.tsx]: In order to make bold styles you should use CSS 👍

**Review by AdamKniec — APPROVED** [review]

**Pull request merged**